src/app/component/eco-news/components/create-news/create-news.component.ts

```diff
@@ -62,6 +62,12 @@ export class CreateNewsComponent implements OnInit {
     }
   }
 
+  public autoResize(event) {
+    event.target.scrollHeight > 100 ?
+      event.target.style.height = '131px' : event.target.style.height = '48px';
+    event.target.style.height = event.target.scrollHeight + 'px';
+  }
```

Comment:
What does it mean? This line certainly rewrite the previous result!

Reply:
fixed